src/elements/Button/Button.module.css

```diff
@@ -165,7 +165,7 @@
       color: var(--button-color);
 
       &:hover {
-        color: var(--button-color-hover);
```

Comment:
why are you removing the hover? it's used for the hover state. you shoudl jsut change the css variable in your repo